src/pudl/metadata/constants.py

```diff
@@ -186,8 +186,17 @@
         "organization": "Catalyst Cooperative",
         "orcid": "0000-0001-9784-8531",
     },
+    "vibrant-clean-energy": {
+        "title": "Vibrant Clean Energy",
+        "email": "info@vibrantcleanenergy.com",
+        "path": "https://vibrantcleanenergy.com/",
+        "role": "author",
+        "zenodo_role": "producer",
+        "organization": "Vibrant Clean Energy",
+    },
```

Comment:
Once we've got an archiver that works for a private file handoff, it'd be good to re-archive the GridPath RA Toolkit data using it, and do something similar for attribution / licensing there.

Reply:
@e-belfer

Reply:
Definitely, though slightly tricky to refresh the metadata without new files but it can be done. Want to write up an issue so it doesn't get lost?

Reply:
See #3895